src/AltinnCore/Designer/TypedHttpClients/TypedHttpClientRegistration.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using AltinnCore.Common.Configuration;
+using AltinnCore.Common.Constants;
+using AltinnCore.Common.Helpers;
+using AltinnCore.Common.Services.Implementation;
+using AltinnCore.Common.Services.Interfaces;
+using AltinnCore.Designer.Infrastructure.Models;
+using AltinnCore.Designer.TypedHttpClients.AltinnStorage;
+using AltinnCore.Designer.TypedHttpClients.AzureDevOps;
+using AltinnCore.Designer.TypedHttpClients.DelegatingHandlers;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace AltinnCore.Designer.TypedHttpClients
+{
+    /// <summary>
+    /// Contains extension methods to register typed http clients
+    /// </summary>
+    public static class TypedHttpClientRegistration
+    {
+        /// <summary>
+        /// Sets up and registers all typed Http clients to DI container
+        /// </summary>
+        /// <param name="services">The Microsoft.Extensions.DependencyInjection.IServiceCollection for adding services.</param>
+        /// <param name="config">The Microsoft.Extensions.Configuration.IConfiguration for </param>
+        /// <returns></returns>
```

Comment:
Add a line about what it returns or remove the line

Reply:
Added comment about what it returns.